benchconnect/benchconnect/utils.py

```diff
@@ -0,0 +1,48 @@
+from json import dumps, load, loads
+from pathlib import Path
+from typing import Any, Dict, List
+
+import click
+
+ENV_VAR_HELP = """
+
+\b
+Environment variables:
+  CONBENCH_URL: The URL of the Conbench server. Required.
+  CONBENCH_EMAIL: The email to use for Conbench login. Only required if the server is private.
+  CONBENCH_PASSWORD: The password to use for Conbench login. Only required if the server is private.
+"""
+
+
+def load_json(json: str, path: str, ndjson: str) -> List[Dict[str, Any]]:
+    "Load JSON from a string, file path, directory path, or stdin"
+
+    stdin = click.get_text_stream("stdin")
+    if not ndjson and not stdin.isatty():
+        ndjson = stdin.read()
+
+    if json:
+        return [loads(json)]
+
+    elif ndjson:
+        return [loads(blob) for blob in ndjson.strip().splitlines()]
+
+    elif path and Path(path).resolve().is_file():
```

Comment:
The latter handles bash-y things, so I can't drop that. I feel uncomfortable dropping the inline one here because the two are far apart and I'm sure I'm going to pass an unresolved path to this accidentally in the future and not discover it till it explodes months later on somebody else's computer. Maybe I am just scarred.

Reply:
ha that's cool!